### PR TITLE
[GAMERENT-1] User, Role, UsersRole entities added, Liquibase changelo…

### PR DESCRIPTION
**Pull request by AleksandrMikhalevich**
User, Role, UsersRole entities added, Liquibase changelog for this entities added

### CODE REVIEW

File: src/main/java/ru/aston/gamerent/model/entity/User.java
Comment (WarmOrange26) [inline src/main/java/ru/aston/gamerent/model/entity/User.java:1]: составные поля в @Column обозначить

File: src/main/resources/db.changelog/changelogs/GAMERENT-1-25-06-2023-changelog.yaml
Comment (WarmOrange26) [inline src/main/resources/db.changelog/changelogs/GAMERENT-1-25-06-2023-changelog.yaml:1]: constraint unique для user email, role name

File: src/main/java/ru/aston/gamerent/model/entity/UsersRole.java
Comment (WarmOrange26) [inline src/main/java/ru/aston/gamerent/model/entity/UsersRole.java:1]: Удалить

File: src/main/java/ru/aston/gamerent/model/entity/Role.java
[the remark this answers is not in the document]
Comment (AleksandrMikhalevich) [reply src/main/java/ru/aston/gamerent/model/entity/Role.java:1]: Done

File: src/main/java/ru/aston/gamerent/model/entity/User.java
Comment (AleksandrMikhalevich) [reply src/main/java/ru/aston/gamerent/model/entity/User.java:1]: Done

File: src/main/resources/db.changelog/changelogs/GAMERENT-1-25-06-2023-changelog.yaml
Comment (AleksandrMikhalevich) [reply src/main/resources/db.changelog/changelogs/GAMERENT-1-25-06-2023-changelog.yaml:1]: Done

File: src/main/java/ru/aston/gamerent/model/entity/UsersRole.java
Comment (AleksandrMikhalevich) [reply src/main/java/ru/aston/gamerent/model/entity/UsersRole.java:1]: Deleted

**Pull request merged**